lib/pdk/module/metadata.rb

```diff
@@ -12,50 +12,50 @@ class Metadata
         'RedHat based Linux' => [
           {
             'operatingsystem'        => 'CentOS',
-            'operatingsystemrelease' => ['7'],
+            'operatingsystemrelease' => ['8'],
           },
           {
             'operatingsystem'        => 'OracleLinux',
-            'operatingsystemrelease' => ['7'],
+            'operatingsystemrelease' => ['8'],
           },
           {
             'operatingsystem'        => 'RedHat',
-            'operatingsystemrelease' => ['7'],
+            'operatingsystemrelease' => ['8'],
           },
           {
             'operatingsystem'        => 'Scientific',
-            'operatingsystemrelease' => ['7'],
+            'operatingsystemrelease' => ['8'],
```

Comment:
The will be no Scientific Linux 8. See https://listserv.fnal.gov/scripts/wa.exe?A2=SCIENTIFIC-LINUX-ANNOUNCE;11d6001.1904